<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Reset default Qt styles */
* {
    margin: 0;
    padding: 0;
    border: 0;
    font: inherit;
}

/* Set a global font for your application */
body {
    font-family: Helvetica, Arial, sans-serif;
}

/* Style for main application window */
#mainWindow {
    background-color: #f4f4f4;
    padding: 10px;
}

/* Style for buttons */
QPushButton {
    background-color: #3498db;
    color: #fff;
    padding: 10px 20px;
    border: none;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #2980b9;
}

/* Style for text input fields */
QLineEdit {
    padding: 8px;
    border: 1px solid #ccc;
    border-radius: 5px;
}

/* Style for labels */
QLabel {
    font-weight: bold;
    color: #333;
}

/* Style for list widgets */
QListWidget {
    border: 1px solid #ccc;
    border-radius: 5px;
    background-color: white;
}

QListWidget::item {
    padding: 10px;
}

QListWidget::item:hover {
    background-color: #f0f0f0;
}

/* Style for progress bars */
QProgressBar {
    text-align: center;
}

/* Style for checkboxes */
QCheckBox {
    padding: 5px;
}

/* Style for radio buttons */
QRadioButton {
    padding: 5px;
}

/* Style for tabs */
QTabWidget::pane {
    border: 1px solid #ccc;
    border-top: none;
    border-radius: 5px;
    background-color: white;
}

QTabBar::tab {
    padding: 10px 20px;
    border: 1px solid #ccc;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
}

QTabBar::tab:selected {
    background-color: #3498db;
    color: #fff;
    border: 1px solid #2980b9;
}

/* Style for menus and menu items */
QMenu {
    background-color: white;
    border: 1px solid #ccc;
}

QMenu::item {
    padding: 8px 16px;
}

QMenu::item:selected {
    background-color: #3498db;
    color: #fff;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QRadioButton" name="sine_radio_Button">
               <property name="text">
                <string>Sine</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="cosine_radio_Button">
               <property name="text">
                <string>Cosine</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Amplitude</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mixer_amplitude"/>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Freuency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mixer_frequency"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Phase Shift</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mixer_phase_shift"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Vertical Shift</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mixer_vertical_shift"/>
           </item>
           <item>
            <widget class="QPushButton" name="mixer_add_btn">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
        <item>
         <widget class="QWidget" name="mixer_widget" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 255, 127);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>193</width>
             <height>284</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="mixer_total_eqn">
          <property name="text">
           <string>Total eqn</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="mixer_done_btn">
          <property name="text">
           <string>Done</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
